main.tf

```diff
@@ -1,8 +1,14 @@
-# ---------------------------------------------------------------------------------------------------------------------
-# CLUSTER KMS KEY
-# ---------------------------------------------------------------------------------------------------------------------
+locals {
+  cluster_encryption_config = length(var.cluster_encryption_config) > 0 ? [
```

Comment:
Hi @bryantbiggs , I believe this conditional should evaluate if the `cluster_encryption_config` variable is not provided which means `== 0` not `> 0`. The default value is an empty list `[]`.

Reply:
ah yes,  you are correct. let me fix this

Reply:
PR https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/1137

Thank you for catching this @muram